apps/nextjs/codegen.ts

```diff
@@ -1,16 +1,28 @@
 import type {CodegenConfig} from '@graphql-codegen/cli';
+// due to issues with TurboRepo and dev dependencies, we can't use this in the monorepo. But if we could, then it would look like this import, and then used below in "scalars"
```

Comment:
```suggestion
// Due to issues with TurboRepo and dev dependencies, we can't use this in the monorepo. But if we could, then it would look like this import, and then used below in "scalars"
```